### ISSUE TITLE
Velocidad de sigilo del pícaro bug

### ISSUE THREAD

**Issue by Saant** [athenaswow/BugTracker-Athenas-WoW#305]
Buenos días. Se supone que al entrar en sigilo, el pícaro reduce su velocidad de movimiento un 40% (así es en Cataclismo), pero esto no sucede, la velocidad sigue normal.  Mando video donde muestro la velocidad y muestro que no tengo ningún talento puesto. En el siguiente video muestro que al poner el talento de Acechador Nocturno (que aumenta la velocidad un 5%) la velocidad sí aumenta (es decir sí funciona correctamente) pero como el Sigilo está dañado, la velocidad no sube a 65% sino a 105%. Esto haría que al poner los 2 talentos la velocidad del pícaro sea de 110% lo cual es un bug muy grande. 
Pj principal: Shadowr
Discord: Saant_ #3848
https://youtu.be/bpQNZdADeLg
https://youtu.be/HTtSYRylLMA
https://cata-twinhead.twinstar.cz/?spell=77780

**Comment by joma997** [comment]
El pícaro se mueve sin restricción en sigilo, al que se le restringe es al dudu

**Comment by Saant** [comment]
> El pícaro se mueve sin restricción en sigilo, al que se le restringe es al dudu

No, el pícaro sí tiene restricción de movimiento, por eso puse este link: https://cata-twinhead.twinstar.cz/?spell=77780 .
Si no tuviera restricción de movimiento entonces el talento de acechador nocturno no tendría el aumento de velocidad.
